cmd/node/config/config.toml

```diff
@@ -416,6 +416,15 @@
     Type = "TxCache"
     Shards = 16
 
+[UserTxDataPool]
+    Name = "UserTxDataPool"
+    Capacity = 600000
+    SizePerSender = 20000
+    SizeInBytes = 419430400 #400MB
```

Comment:
I think this is too much, considering that only API observers will ever use it

Reply:
only duplicated the value for regular tx pool